Formula/qtox.rb

```diff
@@ -0,0 +1,23 @@
+require "formula"
+
+class Qtox < Formula
+  homepage "https://github.com/tux3/qTox"
+  head "git://github.com/tux3/qTox", :using => :git
+  
+  depends_on "Tox/tox/libtoxcore"
+  depends_on "qt5"
+  depends_on "homebrew/science/opencv"
+  depends_on "ReDetection/qtox/libfilteraudio"
```

Comment:
Should be "Tox/tox/libfilteraudio" or "libfilteraudio" since we're merging into homebrew-tox.